packages/tooling/fast-tooling-react/src/form/controls/control.section-link.tsx

```diff
@@ -44,7 +44,7 @@ class SectionLinkControl extends React.Component<
                 )}
                 onClick={this.handleUpdateSection}
             >
-                Edit: {this.props.label}
+                {this.props.strings.sectionLinkEditLabel}: {this.props.label}
```

Comment:
This is an interesting problem with the `:`, I think we can leave it as-is and wait for feedback but I'm wondering how we can deal with a separator in other languages.

Reply:
This would generally be handled by a string formatter, where the code would look like: `format(this.props.strings.sectionLinkEditLabel, this.props.label)` and the injected string might look like `"Edit: {0}"`, where `{0}` is replaced by the second argument to the formatter.

Reply:
We currently have a `format` function in `@microsoft/fast-web-utilitities` that works for this purpose.

Reply:
Done